src/views/Combine.vue

```diff
@@ -0,0 +1,281 @@
+<template>
```

Comment:
this is rather different from the other views. This view:
- knows about breaches
- provides custom interactivity
- loads extra layers
- has different buttons